.github/workflows/vettools/vettools.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 The Ebiten Authors
+// Copyright 2022 The Ebiten Authors
```

Comment:
You don't have to change the year as this is just an existing file.

Reply:
gotcha